### PR TITLE
First working version of logging system. 

### PR DESCRIPTION
**Pull request by gmatoshp**
includes a mock service, fluentd,elasticsearch,and kibana. all services run on their own containers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
 # he-logger
 Hubot Enterprise Audit and Logging System
+This is the very first version with a full working ecosystem. It includes a mock service which generates two logs, one on startup and one when a user points a browser at localhost:3000. The log is sent to the fluentd container, which aggregates it, and sends it to the elasticsearch container. The logs can be seeing by accessing the kibana service container(by default running on localhost:5601).
```
Comment (rqc) [inline README.md]: - [ ] fix typo: logs can be **seen**

Comment (rqc) [inline README.md]: - [ ] add space between container and (

File: README.md [hunk 2]
```diff
@@ -1,2 +1,11 @@
 # he-logger
```
Comment (rqc) [inline README.md]: - [ ] Add backticks to names like `elasticsearch` and `localhost:5601`

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        - HTTP_PROXY=$HTTP_PROXY
+        - HTTPS_PROXY=$HTTPS_PROXY
+        - FTP_PROXY=$FTP_PROXY
+        - NO_PROXY=$NO_PROXY
+    depends_on:
+      - logger
+    ports:
+      - "3000:3000"
+    environment:
+      - NO_PROXY=elastic,logger,localhost,127.0.0.1
```
Comment (rqc) [inline docker-compose.yml]: I thought this needed to be removed... I'm confused now!

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+esdata/
+data/
+myservice/node_modules/
```
Comment (rqc) [inline .gitignore]: - [ ] add newline char

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    environment:
+      - ELASTICSEARCH_URL=http://elastic:9200
+      - NO_PROXY=elastic,localhost,127.0.0.1
+      - no_proxy=elastic,localhost,127.0.0.1
+      - HTTP_PROXY=$HTTP_PROXY
+      - http_proxy=$HTTP_PROXY
+      - HTTPS_PROXY=$HTTP_PROXY
+      - https_proxy=$HTTP_PROXY
+      - FTP_PROXY=$HTTP_PROXY
+      - ftp_proxy=$HTTP_PROXY
```
Comment (rqc) [inline docker-compose.yml]: - [ ] add newline char

File: fluentd/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,3 @@
+FROM fluent/fluentd:v0.14.8
+
+RUN gem install fluent-plugin-elasticsearch
```
Comment (rqc) [inline fluentd/Dockerfile]: - [ ] add newline char

File: myservice/package.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "myservice",
+  "version": "0.0.0",
+  "description": "a test service for the logger",
+  "main": "server.es6",
+  "dependencies": {
```
Comment (rqc) [inline myservice/package.json]: - [ ] verify that these are the only deps by removing `/node_modules` and running `npm install` again and re-running everything.

File: docker-compose.yml [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+      - NO_PROXY=elastic,logger,localhost,127.0.0.1
+      - no_proxy=elastic,logger,localhost,127.0.0.1
+      - HTTP_PROXY=$HTTP_PROXY
+      - http_proxy=$HTTP_PROXY
+      - HTTPS_PROXY=$HTTP_PROXY
+      - https_proxy=$HTTP_PROXY
+      - FTP_PROXY=$HTTP_PROXY
+      - ftp_proxy=$HTTP_PROXY
+    volumes:
+      - ./myservice:/usr/src/app
```
Comment (rqc) [inline docker-compose.yml]: good!! 👍  No more re-building!

File: myservice/server.es6 [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+  fluent.emit('hello', {record: 'hello world was sent'});
+  console.log("trying to emit hello world message");
+
+});
+
+app.listen(3000, function () {
+  console.log('Example app listening on port 3000 ftw!');
+  fluent.emit('startup', {record: 'auth service started propery'});
+
+});
```
Comment (rqc) [inline myservice/server.es6]: - [ ] add newline char

File: myservice/server.es6 [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+//configure the logger, dummy values for now, will change to real values afterwards
+
+fluent.configure('myservice', {
+  host: 'logger',
+  port: 24224,
+  timeout: 3.0,
+  reconnectInterval: 600000 //10 minutes
+});
+
+/* eslint-disable camelcase */
```
Comment (rqc) [inline myservice/server.es6]: - [ ] remove comment

File: docker-compose.yml (hunk 3 above)
Comment (gmatoshp) [inline docker-compose.yml]: you are right. removed them all. the ecosystem works as expected.

**Pull request merged**